README.md

```diff
@@ -260,31 +209,28 @@ Also, we are considering extensions to this proposal.  These extensions include:
   both.
 
 Any of these new features, if implemented, would require a different API shape
-than what is currently in the spec.  Therefore, the current pattern is not very
+than what is currently in the spec. Therefore, the current pattern is not very
 good for extensibility along these lines.
 
 ### Setting
 
-This exposes the current blur state as a boolean.  Our proposal uses an enum,
+This exposes the current blur state as a boolean. Our proposal uses an enum,
 which is preferred for extensibility.
 
-Our proposal goes beyond this and adds an event for when the blur status
-changes.  The current spec does not provide applications a way to know when the
-current blur state has changed.
-
 ### Summary
 
 Features of our proposal missing from current spec:
 
-- Events for effects status changes.
+- Per-frame information about effect state
 - Extensibility for future ways applications could interact with effects.
 
 Features of current spec missing from our proposal:
 
 - Ability to know whether effects can be turned on or off.
 
-Below is an [extension of the current proposal](#exposing-ability-for-applications-to-turn-blur-on-or-off)
-that would add this feature.
+Below is an [extension of the current
+proposal](#exposing-ability-for-applications-to-turn-blur-on-or-off) that would
+add this feature.
```

Comment:
We can drop this from the doc as we're not going to be making any additions to the track API in the current round. 

```suggestion
```